Makefile

```diff
@@ -72,6 +72,21 @@ coverage:
 	@ADMIN_USERNAME="" ADMIN_PASSWORD="" CONFIGMAP_NAME="" DB_SERVER="" PORTAL_URL=""
 
 
+.PHONY: golint
+golint:
+	# TODO: Ditch golint in favour of revive, see https://revive.run for more details
+	@echo "------------------"
+	@echo "--> Running GoLint"
+	$(eval PKGS := $(shell go list ./... | grep -v /vendor/))
+	@touch golint.tmp
+	-@for pkg in $(PKGS) ; do \
+		echo `golint $$pkg | grep -v "have comment" | grep -v "comment on exported" | grep -v "lint suggestions"` >> golint.tmp ; \
+	done
+	@grep -Ev "^$$" golint.tmp || true
```

Comment:
Is this needed as you are semantically checking the same thing in next line?

Reply:
This line is outputting the non excluded suggestions/warnings, the latter line is just giving us the right message and the right exit code for the shell, this won't be required if we just switch to revive or use https://golangci-lint.run/ to run revive among a host of other nice tools.